src/file/FileControllerNative.ts

```diff
@@ -24,30 +25,62 @@ export class FileControllerNative implements FileController {
 	 * Temporary files are deleted afterwards in apps.
 	 */
 	async download(file: TutanotaFile) {
-		await guiDownload(this.doDownload(file))
+		await guiDownload(
+			isAndroidApp() || isDesktop()
+				? this.doDownload(
+						new Array(file),
```

Comment:
we can just do [file], `new Array()` has this weird overload where it's creating an array of length n if n is a number and the only parameter

Reply:
already resolved in the latest commit :+1: